### PR TITLE
LPS-121827 set field reference initial value when adding an element set

### PR DESCRIPTION
**Pull request by marcosapmf**
Related Issue: https://issues.liferay.com/browse/LPS-121827

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @marcosapmf, did you have a chance to analyse if we could create some tests for this pull?

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/util/fieldSupport.es.js [hunk 1]
```diff
@@ -68,7 +69,7 @@ export const createField = (props, event) => {
 		},
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/util/fieldSupport.es.js:69]: @marcosapmf eu juntei com a parte que eu adiciono o Field Reference ([marcosapmf#239](https://github.com/marcosapmf/liferay-portal/pull/239)) e parece que só isso não funciona. Pelo que eu entendi, tem que atualizar o FieldReference no `normalizeSettingsContextPages ` tambem.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/elementSetAddedHandler.es.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const handleElementSetAdded = (props, state, event) => {
 		return {
 			...field,
 			fieldName: name,
+			fieldReference: name,
 			settingsContext: {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/elementSetAddedHandler.es.js:37]: Acredito que o `settingsContext` deve ser atualizado aqui tambem

**Comment by carolmariaabb** [comment]
@marcosapmf seria bom ver a necessidade de atualizar no `fieldDuplicatedHandler.es`. Ele atualiza o fieldName la
`	let duplicatedField = updateField(
		props,
		originalField,
		'name',
		newFieldName
	);`

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/elementSetAddedHandler.es.js (hunk 2 above)
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/elementSetAddedHandler.es.js:37]: Certo, @carolmariaabb, eu vou dar uma olhada. Valeu! :D

**Pull request closed without merging**